client/src/modules/reports/generate/income_expense_by_month/income_expense_by_month.config.js

```diff
@@ -0,0 +1,65 @@
+angular.module('bhima.controllers')
+  .controller('income_expense_by_monthController', IncomeExpenseByMonthConfigController);
+
+IncomeExpenseByMonthConfigController.$inject = [
+  '$sce', 'NotifyService', 'BaseReportService', 'AppCache', 'reportData', '$state',
+];
+
+function IncomeExpenseByMonthConfigController($sce, Notify, SavedReports, AppCache, reportData, $state) {
+  const vm = this;
+  const cache = new AppCache('configure_income_expenseBy_month');
```

Comment:
For consistency, use `configure_income_expense_by_month`, since that is the URL.